contentcuration/contentcuration/utils/websocket_helper.py

```diff
@@ -0,0 +1,91 @@
+import uuid
+
+from le_utils.constants import content_kinds
+
+from contentcuration import models
+from contentcuration.models import Change
+from contentcuration.tests.viewsets.base import generate_create_event
+from contentcuration.tests.viewsets.base import generate_update_event
+from contentcuration.viewsets.sync.constants import BOOKMARK
+from contentcuration.viewsets.sync.constants import CHANNEL
+from contentcuration.viewsets.sync.constants import CONTENTNODE
+from contentcuration.viewsets.sync.constants import EDITOR_M2M
+
+
+class NoneCreatedByIdError(Exception):
+    """
+    Use to log change object whose created_by_id is set to none. We don't raise this error,
+    just feed it to Sentry for reporting.
+    """
+
+    def __init__(self, instance):
+
+        self.change_object = instance
+        message = (
+            "The change object did not have a created_by_id {}"
+        )
+        self.message = message.format(
+            instance.__dict__
+        )
+
+        super(NoneCreatedByIdError, self).__init__(self.message)
+
+
+def bookmark_metadata(channel):
```

Comment:
In my last review, I didn't notice that this file lives outside of the `tests/` directory. To keep a clear separation between test utilities and code needed for core functionality, I think it would be best to relocate everything below to inside `tests/`